media.css

```diff
@@ -0,0 +1,125 @@
+@media (max-width: 1550px) {
+  /*.other {
+    display: flex;
+    flex-direction: column;
+  }*/
+
+  
+  .center__v {
+    padding-top: 100px;
+  }
+}
+
+  
+@media (max-width: 1453px) {
+  .middle__title {
+    right: 0px;
+    top:150px;
+  }
+
+  .content {
+    left: 0;
+    bottom: 0;
+    padding-left: 500px;
+    padding-right: 500px;
+  }
+
+  .container {
+  max-width: 100%;
+  bottom: 0px;
+  top: 900px;
+  }
+  .underside {
+  bottom: 1000px;
+  left: 0px;
+  }
+}
+
+
+@media (max-width: 1335px) {
+	.menu__link_modelS {
+		display: none;
+	}
+
+  .header__line {
+    opacity: 0;
+  }
+
+  .top__subtitle {
+   
+  }
+}
+
+@media (max-width: 1202px) {
+  .menu__link_modelX {
+    display: none;
+  }
+}
+
+@media (max-width: 1114px) {
+  .menu__link_model3 {
+    display: none;
+  }
+}
+
+@media (max-width: 1006px) {
+  .menu__link_roadster {
+    display: none;
+  }
+
+  .header__burger {
+    position: absolute;
+    width: 35.23px;
+    height: 24px;
+    background: url(./images/icon_burg.png);
+  }
+
+.header__logo {
+  margin-left: 0;
+  }
+
+header {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)